setup.py

```diff
@@ -12,6 +12,7 @@
     author_email='jarbasai@mailfence.com',
     license='Apache-2.0',
     packages=['ovos_vad_plugin_silero'],
+    package_data={'ovos_vad_plugin_silero': ['silero_vad.onnx']},
```

Comment:
Would we ever include a differently named model file here? We could probably include `*.onnx` in package_data if we might, but if  its just a static model then I think this is fine

Reply:
if we later add support for more models we can do that, for now only this one exists